File: WebLeave/API/Dtos/Leave/Personal/PersonalDataViewModel.cs

namespace API.Dtos.Leave.Personal
{
    public class PersonalDataViewModel
    {
        public List<LeaveDataViewModel> LeaveDataViewModel { get; set; }
        public EmployeeDataDto Employee { get; set; }
        public HistoryEmployeeDto History { get; set; }
    }
}

File: WebLeave/SPA/src/app/_core/models/leave/personal/personalDataViewModel.ts
import { LeaveDataViewModel } from "../leaveDataViewModel";
import { EmployeeData } from "./employeeData";
import { HistoryEmployee } from "./historyEmployee";

export interface PersonalDataViewModel {
    leaveDataViewModel: LeaveDataViewModel[];
    employee: EmployeeData;
    history: HistoryEmployee;
}